src/commands/start.js

```diff
@@ -43,10 +50,29 @@ class StartCommand extends BaseCommand {
             CORE_MASTERNODE_BLS_PRIV_KEY = '';
           }
 
+          let COMPOSE_FILE = '';
+
+          if (driveSrcPath || dapiSrcPath) {
```

Comment:
We shouldn't allow these options for testnet preset until we land the platform there